.circleci/config.yaml

```diff
@@ -0,0 +1,40 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: cimg/python:3.7
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - deps-{{ checksum "poetry.lock" }}
+
+      - run:
+          name: Install Dependencies
+          command: poetry install
+
+      - save_cache:
+          key: deps-{{ checksum "poetry.lock" }}
+          paths:
+            - /home/circleci/.cache/pypoetry/virtualenvs
+
+      - run:
+          name: Run pylint
+          command: poetry run pylint symphony_api_client_python tests > pylint_report.txt
+
+      - run:
+          name: Run tests with coverage
+          command: |
+            mkdir test-results
```

Comment:
is it needed? (could it be part of the build? to avoid having to do it locally or maybe locally we would just run without args and use the console?)